.travis.yml

```diff
@@ -0,0 +1,17 @@
+language : node_js
+
+node_js: node_js
+
+cache: npm

```

Comment:
`cache`: `node_modules`

Reply:
error on my part, accidentally filled out npm instead of node_modules